internal/vds/vds.go

```diff
@@ -327,9 +327,9 @@ func GetHorizon(
 	for i := range data {
 		if len(data[i]) != ncols  {
 			msg := fmt.Sprintf(
-				"Invalid coordinate %v at position %d, expected [x y] pair",
-				data[i],
-				i,
+				"Invalid coordinate at position %d. "+
+					"Expected all rows to have length %d, was %d",
```

Comment:
We didn't expect rows of a particular length. Rather we expect them all to be _the same_ length